packages/channels/src/base/stream.ts

```diff
@@ -15,6 +15,8 @@ export abstract class ChannelStream<TService extends ChannelService<any, any>, T
   }
 
   protected async handleSend({ scope, endpoint, content }: ChannelSendEvent) {
+    await this.service.require(scope)
+
```

Comment:
Basically this was needed to be done to make the auto start from bp -> channel work